asciidoc-release-notes/adoc/attributes-generic.adoc

```diff
@@ -220,4 +220,16 @@
 :ultrascalereg: {ultrascale}*
 :zynq: Zynq
 :zynqreg: {zynq}*
-
+// Container-related attributes
+:podman: Podman
+:buildah: Buildah
+:docker: Docker
+:deng: {docker} Open Source Engine
+:dhub: {docker} Hub
+:dreg: {docker} Registry
+:bci: Base Container Image
+:bcia: BCI
+:kiwi: KIWI
+:rmt: RMT
+:rmtool: Repository Mirroring Tool
```

Comment:
To keep with the convention I would call these two :rmta: and :rmt:, respectivelly:
```suggestion
:rmta: RMT
:rmt: Repository Mirroring Tool
```

Reply:
Actually, `&rmt;` and `&rmtool;` are used throughout the container guide (and in the `generic-entities.ent` file). So if you don't mind I'd like to keep them as they are to avoid manually changing them  and thus risking breaking stuff.

Reply:
Argh! 😅 I'll have to hunt down whoever created those... Let's keep it as-is for now, then.